dev/index.js

```diff
@@ -1,73 +1,291 @@
+        let span = $("<span/>").attr({
+          "data-i18n": `[html]about_us.vision.contents.${i}.${j}`
+        });
+        li.append(icon);
+        li.append("&nbsp;");
+        li.append(span);
+        ulElem.append(li);
+      }
+      $("#visionList").append(ulElem);
+    } else {
+      $("#visionList").append(
+        "<p data-i18n='[html]about_us.vision.contents." + i + "'></p>"
+      );
     }
-}
-*/
+  }
+  /* End vision section */
 
-function open_url_tab(url) {
-    var win = window.open(url, '_blank');
-    if (win) { // Browser has allowed it to be opened
-        win.focus();
-    } else { // Browser has blocked it
-        alert('Please allow popups for this website');
+  /* Strat team section */
+  for (let i = 0; i < 14; i++) {
+    let outerDiv = $("<div/>")
+      .addClass("col-lg-2 col-md-6")
+      .attr({
+        "data-i18n": "[data-content]about_us.teams.list." + i + ".content"
+      });
+    let innerDiv1 = $("<div/>").addClass("member");
+    let innerDiv2 = $("<div/>").addClass("pic");
+    let img = $("<img/>")
+      .attr({
+        src: "../../assets/team/portrait_" + i + ".jpg"
+      })
+      .css({width: "100px", height: "100px"});
+    innerDiv2.append(img);
+    innerDiv1.append(innerDiv2);
+    innerDiv1.append(
+      '<h5 style="font-size:16px" data-i18n="[html]about_us.teams.list.' +
+      i +
+      '.name"></h5>'
+    );
+    innerDiv1.append(
+      '<span data-i18n="[html]about_us.teams.list.' + i + '.type"></span>'
+    );
+    outerDiv.append(innerDiv1);
+    $("#teamList").append(outerDiv);
+  }
+  /* End team section */
+
+  /* Strat contact section */
+  let table = $("<table />").addClass("table table-sm table-bordered");
+  let tbody = $("<tbody />");
+  for (let i = 0; i < 8; i++) {
+    let tr = $("<tr />");
+    /* For First Row */
+    let td1 = $("<td />")
+      .css({width: "30%", height: "40px"})
+      .addClass("text-center align-middle")
+      .attr({
+        "data-i18n": "[html]about_us.contact.contents." + i + ".key"
+      });
+    tr.append(td1);
+    /* For Second Row */
+    let td2 = $("<td />")
+      .css({height: "40px"})
+      .addClass("align-middle")
+      .attr({
+        "data-i18n": "[html]about_us.contact.contents." + i + ".value"
+      });
+    tr.append(td2);
+
+    tbody.append(tr);
+  }
+  table.append(tbody);
+  $("#contactList").append(table);
+  /* End Contact section */
+
+  /* Strat support section */
+  for (let i = 0; i < 4; i++) {
+    $("#supportContents").append(
+      '<span data-i18n="[html]about_us.support.contents.' + i + '"></span><br/>'
+    );
+  }
+  /* End support section */
+
+  /* Strat jobs Section */
+  for (let i = 0; i < 12; i++) {
+    if (i === 5) {
+      let ulElem = $("<ul />");
+      for (let j = 0; j < 5; j++) {
+        let li = $("<li />");
+        li.append(
+          "<i class='fas fa-angle-double-right'></i>&nbsp;<span data-i18n='[html]about_us.jobs.contents." +
```

Comment:
**rewrite all hard coded HTML fragment strings with jQuery EVERYWHERE** (was already the earlier review comment)